modules/ROOT/pages/notifications/index.adoc

```diff
@@ -1,35 +1,203 @@
 :description: The Neo4j notifications for Neo4j version {neo4j-version}.
 
 [[notifications]]
-= Notification codes
+= Server notifications
 
-Notification codes are Neo4j status codes of type `ClientNotification`and are returned by the server with severity `WARNING` or `INFORMATION`.
-Contrary to errors, notifications are only returned if the execution of the query is successful.
+After a successful query execution, the Neo4j server sends notifications to provide advice on how to improve the query's quality or give additional information about the query execution.
+The driver receives these notifications and sends them to the client, which displays them to the user.
 
-[[notification-format]]
-== Notification object fields
+From version 5.23, Neo4j has a new GqlStatusObject API in addition to the existing Notification API.
+The GqlStatusObject API provides information about the status of a Cypher query or command execution in compliance with the GQL standard.
 
-The notification object, which is part of the result summary, consists of the following fields:
+This page describes both the Neo4j Notification and the GQL-status object frameworks, how they are structured, the objects they provide for notifications, and how to interpret them.
```

Comment:
I think we are still not fully consistent on naming. Should we stick to `GqlStatusObject` (this is what it is called in the codebase) when we talk about the framework/API and `GQL-status object` (this is what it is called in GQL standard) if we talk about the actual object a user get back or what do you think? Can you read through and check that it is consistent?